lib/WTSI/DNAP/Warehouse/Schema/Result/IseqRunLaneMetric.pm

```diff
@@ -218,6 +218,14 @@ tag0_perfect_match_reads as a percentage of total_lane_reads
 
 Sequencing lane level run priority, a result of either manual or default value set by core
 
+=head2 pct_tag_hops
```

Comment:
Please can we call this `tag_hops_percent` so that is is consistent with the `tags_decode_percent` and `unexpected_tags_percent` fields here and also with `adapters_percent*` and `gc_percent_*` elsewhere?